src/Makefile

```diff
@@ -93,9 +93,7 @@ endif
 # the user can override with `make ARCH=x86-32-vnni256 SUPPORTED_ARCH=true`
 ifeq ($(ARCH), $(filter $(ARCH), \
                  x86-64-vnni512 x86-64-vnni256 x86-64-avx512 x86-64-bmi2 x86-64-avx2 \
-                 x86-64-sse41-popcnt x86-64-modern x86-64-ssse3 x86-64-sse3-popcnt \
-                 x86-64 x86-32-sse41-popcnt x86-32-sse2 x86-32 ppc-64 ppc-32 \
-                 armv7 armv7-neon armv8 apple-silicon general-64 general-32))
+                 x86-64-sse41-popcnt x86-64-modern x86-64-ssse3))
```

Comment:
I guess that's part of the work in progress right?

Reply:
Yes, I decided to remove the rest of the code for simplicity and only leave what matters here. For merge into master the code for <ssse3 would be included.